pkg/sync/v6/data/PrometheusRule.monitoring.coreos.com/openshift-monitoring/prometheus-k8s-arorules.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: prometheus-k8s-arorules
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: aro-sync.rules
+    rules:
+    - alert: SyncPodDown
+      annotations:
+        description: Sync Pod is down
+        summary: Sync Pod is down
+      expr: |
+        absent(up{service="sync"} == 1)
+      for: 10m
+      labels:
+        severity: warning
+    - alert: SyncNotExecuted
+      annotations:
+        description: There was no Sync in the last 10min
+        summary: Sync not executed in last 10min
+      expr: |
+        time() - sync_last_executed > 600
+      labels:
+        severity: warning
+  - name: aro-canary.rules
+    rules:
+    - alert: CanaryPodDown
+      annotations:
+        description: At least one Canary Pod is down for 10min
+        summary: Canary Pod is down
+      expr: |
+        absent(sum(up{service="canary"}) == 2)
+      for: 10m
+      labels:
+        severity: warning
+  - name: aro-router.rules
+    rules:
+    - alert: RouterPodDown
+      annotations:
+        description: At least one Router Pod is down for 10min
+        summary: Router Pod is down
+      expr: |
+        absent(sum(up{service="router-stats"}) == 3)
+      for: 10m
+      labels:
+        severity: warning
+    - alert: RouterResponseErrors
+      annotations:
+        description: Router sent 5xx responses for {{$labels.route}} pod {{$labels.pod}},
+          exported pod {{$labels.exported_pod}}
+        summary: Router sent 5xx responses on internal services
+      expr: |
+        rate(haproxy_server_http_responses_total{code="5xx",route=~"docker-registry|registry-console|branding|canary|console|alertmanager-main|grafana|prometheus-k8s"}[1m]) > 0
+      labels:
+        severity: warning
+    - alert: RouterDowntime
+      annotations:
+        description: Router noticed downtime of {{$labels.route}} pod {{$labels.pod}}
+        summary: Router noticed downtime
+      expr: |
+        rate(haproxy_server_downtime_seconds_total{route=~"docker-registry|registry-console|branding|canary|console|alertmanager-main|grafana|prometheus-k8s"}[1m]) > 0
```

Comment:
Is this something we want to alert on? This seems like it could be very noisy.

Reply:
It doesn't translate 1:1 to alerts/issues in Geneva. I wanted to send it all as part of the ALERTS metric and then select on what we actually alert.

If it turns out to be too noisy to use in Geneva alerting I'll remove it.